### PR TITLE
add "Other Tips" notes to TESTGUIDE.md and a script to update baseline files under test/fsharp

### PR DESCRIPTION
**Pull request by smoothdeveloper**
Small PR to make contributions easier to test/fsharp

changes:
* helper script to update baseline files under tests/fsharp
* rework "Other Tips" section to expand on the system of baselines files used in FSharp Test Suite

### CODE REVIEW

**Comment by smoothdeveloper** [comment]
@dotnet-bot test Ubuntu16.04 Release_default Build please
@dotnet-bot paket convert-from-nuget please

File: tests/fsharp/update.base.line.with.actuals.fsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+// the script replaces all the .bsl/.bslpp with either .err or .vserr
+
+let directories =
+    [
+      "typecheck/sigs"
+      "typeProviders/negTests"
+    ]
+    |> List.map (fun d -> Path.Combine(__SOURCE_DIRECTORY__, d) |> DirectoryInfo)
+
+let extensionPatterns = ["*.err"; "*.vserr"]
```
Comment (dsyme) [inline tests/fsharp/update.base.line.with.actuals.fsx]: You should also move `*.il` --> `*.il.bsl` under `fsharpqa\Source`, e.g.

image

Comment (dsyme) [reply tests/fsharp/update.base.line.with.actuals.fsx]: Make it a unified script under `tests\scripts`?

**Comment by dsyme** [comment]
This is good, just the one adjustment requested

**Comment by dsyme** [comment]
@smoothdeveloper I'll make the change as I need to update some `fsharpqa` baselines in any case

**Comment by smoothdeveloper** [comment]
Thanks, great!

**Pull request merged**